docs/build/_toc.json

```diff
@@ -47,7 +47,7 @@
       "title": "Advanced circuit options",
       "children": [
         {
-          "title": "Pulse gates",
+          "title": "Tune pulse waveforms",
```

Comment:
Tune pulse waveforms would be okey if this is ad headline like. Strictly speaking, pulse gate is a microcode that defines a calibration of gate, and what it provides is a schedule (pulse sequence) rather than waveform.

Reply:
Hm, how about `Tune pulse schedules`? I think @kaelynj suggested "Tune pulse waveforms", so tagging her if she has more feedback!

Reply:
I think `Tune Pulse Schedules` or `Tune Pulse Instructions` would work well (also maybe "Tuning" instead of "Tune"?).  Mostly I'm thinking of trying to communicate a broad idea to the reader of what Pulse is used fore.

Reply:
Tune is more active than Tuning, but another option would be to remove the verb altogether if it sounds too weird with it in. I got feedback at lunch today to drop Tune from the title. What do we think of just `Pulse schedules`?

Reply:
Let's go with Pulse Schedules :)